src/components/filters/MeasureValueFilter/OperatorDropdown.tsx

```diff
@@ -0,0 +1,109 @@
+// (C) 2007-2019 GoodData Corporation
+import * as React from "react";
+import { injectIntl, InjectedIntlProps } from "react-intl";
+import classNames from "classnames";
+import noop from "lodash/noop";
+import capitalize = require("lodash/capitalize");
+import Button from "@gooddata/goodstrap/lib/Button/Button";
+
+import OperatorDropdownBody from "./OperatorDropdownBody";
+import * as Operators from "./operators";
+import { getOperatorTranslationKey, getOperatorIcon } from "./utils";
+
+interface IOperatorDropdownOwnProps {
+    onSelect: (operator: string) => void;
+    operator: string;
+}
+
+type IOperatorDropdownProps = IOperatorDropdownOwnProps & InjectedIntlProps;
+
+interface IOperatorDropdownState {
+    opened: boolean;
+}
+
+export class OperatorDropdown extends React.PureComponent<IOperatorDropdownProps, IOperatorDropdownState> {
+    public static defaultProps: any = {
+        onSelect: noop,
```

Comment:
Does the component make sense if the user does not provide the `onSelect` handler? Should we provide a default?

Reply:
You are right, I have removed the default value. `onSelect` is mandatory now.